pyephember/pyephember.py

```diff
@@ -190,88 +441,152 @@ def _get_first_gateway_id(self):
             raise RuntimeError("Cannot get gateway id from list of homes.")
         return self._homes[0]['gatewayid']
 
+    def _set_zone_target_temperature(self, zone, target_temperature):
+        return self.messenger.zone_command(
+            zone,
+            [0, PointIndex.TARGET_TEMP.value, 4, 0, int(10*target_temperature)]
```

Comment:
It's not really clear why you have `10*target_temperature` here. Is it some conversion based on the PointData data types?

Reply:
Yes, the units used for the mqtt commands are tenths of a degree.